### PR TITLE
Merge dev to eventhub and remove the use of power-mock

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

**Review by spoorthipujariadobe — APPROVED** [review]

File: code/android-core-library/src/test/java/com/adobe/marketing/mobile/services/NetworkServiceTests.java [hunk 1]
```diff
@@ -1,274 +1,272 @@
...
-            @Override
-            public void execute(Runnable command) {
-                // Executes task in the caller's thread
-                callerRunsPolicy.rejectedExecution(command, this);
-            }
-        };
-    }
-
-}
+//TODO: The tests are updated on eventhub branch, will update this class after the changes are merged.
```
Comment (spoorthipujariadobe) [inline code/android-core-library/src/test/java/com/adobe/marketing/mobile/services/NetworkServiceTests.java:1]: Do you plan on fixing these later?

Comment (yangyansong-adbe) [reply code/android-core-library/src/test/java/com/adobe/marketing/mobile/services/NetworkServiceTests.java:1]: Good catch, I will update it.

File: code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchTokenFinderTest.kt [hunk 2]
```diff
@@ -18,8 +17,9 @@ import com.adobe.marketing.mobile.MobileCore
 import com.adobe.marketing.mobile.SharedStateResult
 import com.adobe.marketing.mobile.SharedStateStatus
 import com.adobe.marketing.mobile.internal.eventhub.EventHub
-import com.adobe.marketing.mobile.internal.utility.TimeUtil
+import com.adobe.marketing.mobile.utils.TimeUtil
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
 import org.junit.Before
```
Comment (spoorthipujariadobe) [inline code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchTokenFinderTest.kt:25]: You fixed this class to not extend from `BaseTest` in your previous PR right? Did it get overwritten while resolving conflicts? Can we remove it?

File: code/android-core-library/src/test/kotlin/com/adobe/marketing/mobile/internal/utility/TimeUtilTest.java [hunk 3]
```diff
@@ -19,6 +19,8 @@
 
 import static org.junit.Assert.*;
 
+import com.adobe.marketing.mobile.utils.TimeUtil;
```
Comment (spoorthipujariadobe) [inline code/android-core-library/src/test/kotlin/com/adobe/marketing/mobile/internal/utility/TimeUtilTest.java:22]: Must have missed moving the tests. Can we move this class to `test/java/com/adobe/marketing/mobile/utils` package?

File: code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchTokenFinderTest.kt (hunk 2 above)
Comment (yangyansong-adbe) [reply code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchTokenFinderTest.kt:25]: Yes, the git merge added it back, let me remove it now.

**Review by praveek — APPROVED** [review]

**Pull request merged**